src/dynamodb/lock.py

```diff
@@ -11,22 +9,10 @@
 lock_client = DynamoDBLockClient(
     dynamodb_resource,
     table_name=LOCK_TABLE,
+    lease_duration=timedelta(seconds=20),
```

Comment:
oooh spicy

Reply:
hmm do we have any way to log if the lease expires but the lock hasn't been released yet, might be helpful for debugging if this is too short